cosmos/operators/local.py

```diff
@@ -132,9 +155,10 @@ def run_subprocess(self, *args: Any, **kwargs: Any) -> FullOutputSubprocessResul
 
     def run_command(
         self,
-        cmd: list[str | None],
+        cmd: list[str],
         env: dict[str, str | bytes | os.PathLike[Any]],
         context: Context,
+        emit_openlineage: bool = False,
```

Comment:
Why `emit_openlineage`? 

If OpenLineage integration exists and is properly configured, it will call `get_openlineage_facets_on_complete` itself. I don't think there's need to have additional explicit boolean check for it, although I don't fully know the context.

Reply:
By default, it seems the common processor would raise an exception if it is an unsupported dbt node:
https://github.com/OpenLineage/OpenLineage/blob/main/integration/common/openlineage/common/provider/dbt/processor.py#L141

I'll change our code to set:
```
    should_raise_on_unsupported_command = False
```